src/battle_game/core/README.md

```diff
@@ -1,15 +1,15 @@
 # Core
 
-这个文件夹下的内容是整个项目的核心框架。
-[GameCore](game_core.h) 类是所有游戏逻辑集中协调的地方。
+����ļ����µ������������Ŀ�ĺ��Ŀ�ܡ�
+[GameCore](game_core.h) ����������Ϸ�߼�����Э��ĵط���
 
-在这个框架中，游戏内容主要分为四类元素：单位（[Units](unit.h)）、障碍物（[Obstacles](obstacle.h)）、子弹（[Bullets](bullet.h)）和粒子（[Particles](particle.h)）。
-这四类元素的基类统一继承自对象类型（[Object](object.h)），他们都有一定特定的预设功能，用于方便你实现自己的想法。
+��������У���Ϸ������Ҫ��Ϊ����Ԫ�أ���λ��[Units](unit.h)�����ϰ��[Obstacles](obstacle.h)�����ӵ���[Bullets](bullet.h)������ӣ�[Particles](particle.h)����
+������Ԫ�صĻ���ͳһ�̳��Զ������ͣ�[Object](object.h)�������Ƕ���һ���ض���Ԥ�蹦�ܣ����ڷ�����ʵ���Լ����뷨��
```

Comment:
Lots of meaningless changes in this file (maybe comes from formatter), just fix this, don't commit this changes